src/utils/airtable.js

```diff
@@ -43,3 +43,29 @@ export default function getComponentDetails(componentName) {
       );
   });
 }
+
+// this function should return title and image data for all components
+export async function getHomePageInfo() {
+  // connect to our table 'Components
+  const records = await base('Components').select().firstPage();
+  const homePageInfo = records.map((record) => {
+    if (!record.fields['Default Image']) {
+      // Add coming soon image
+    }
+    const obj = {};
+    obj[record.fields['Component Name']] = {
+      name: record.fields['Component Name'],
+      defaultImage: record.fields['Default Image'][0].url,
+      hoverImage: record.fields['Hover Image'][0].url,
+    };
+    return obj;
+  });
+
+  const homePageObject = {};
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < homePageInfo.length; i++) {
```

Comment:
I don't feel strongly about `no-plusplus` as a rule, but rather than disabling it with a comment, can we either disable that rule globally or switch this to be `i += 1`?

Reply:
After looking at #162, this is going to go away anyway, so don't sweat it 😄